models/cex/cex_addresses.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(
+        alias ='addresses',
+        post_hook='{{ expose_spells(\'["ethereum", "bnb", "avalanche_c", "optimism", "arbitrum", "polygon", "bitcoin", "fantom"]\',
+                                    "sector",
+                                    "cex",
+                                    \'["hildobby"]\') }}')
+}}
+
+
+{% set nft_models = [
```

Comment:
I will change the list name.
```suggestion
{% set cex_models = [
```